### ISSUE TITLE
Generic build error when updating from 10.2.4-canary.2 to any version above

### ISSUE THREAD

**Issue by huggge** [vercel/next.js#26574]
### What version of Next.js are you using?

10.2.4-canary.3

### What version of Node.js are you using?

14.16.1

### What browser are you using?

Chrome

### What operating system are you using?

MacOs & Windows

### How are you deploying your application?

next start

### Describe the Bug

When using 10.2.4-canary.2 or earlier versions building works fine, but when upgrading to 10.2.4-canary.3 or any version above I get this very anonymously error. It do however work to start in dev

`

    yarn build                        
    yarn run v1.22.10
    $ next build
    info  - Loaded env from /Users/user/Desktop/dashboard/.env.local
    info  - Using webpack 4. Reason: custom webpack configuration in next.config.js https://nextjs.org/docs/messages/webpack5
    NODE ENV: production
    info  - Checking validity of types  
    To use ESLint, additional required package(s) must be installed.
    
    Please install eslint-config-next by running:
    
            yarn add --dev eslint-config-next
    
    If you do not want to run ESLint during builds, run next build --no-lint or remove the .eslintrc.json file from your package root.
    
    error Command failed with exit code 1.
    info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

`

This is happening in two separate projects and I have tried building them on both macOs and Windows, but dont even know where to start debugging with this very generic build error.

Does anyone have a clue of what could be causing this? I've narrowed it down to 10.2.4-canary.3

Lord please..!

### Expected Behavior

Should be able to build project with never version than 10.2.4-canary.2

### To Reproduce

Update to 10.2.4-canary.3 and try to build

**Comment by PythonCreator27** [comment]
@huggge Either run `yarn add -D eslint-config-next` (to add next.js eslint config), or run `yarn build --no-lint` to not run linting on builds. Or, lastly, remove the eslint config from your package root.

**Comment by housseindjirdeh** [comment]
@PythonCreator27 is correct, you'll have to do one of the following:

- Install `eslint-config-next` and it to your config like @PythonCreator27 mentioned
- Disable ESLint from running in builds using `ignoreDuringBuilds` in your next config (See: https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint)
- Run builds using the `--no-lint` flag (i.e.`next build --no-lint`)

Note: I have a [PR](https://github.com/vercel/next.js/pull/26584) open that improves the developer experience and does not require `eslint-config-next` to be installed. Once that lands, you'll never see this error message again - but in the meantime you'll have to do one of the above :)

**Comment by huggge** [comment]
Thanks for the help guys!

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
